### DISCUSSION TITLE
Static Optimization Indicator not showing up for dynamic routes

### DISCUSSION THREAD

**Discussion by nickmarca** [vercel/next.js#16609] [Help] [answered]
Hello. 

Note: This is a very minor bug and  I'm not fully convinced this is actually a bug or if I am misunderstanding how it works.

**Bug description:**

Static Optimization Indicator is not showing up for pages which are qualified for Automatic Static Optimization but include Dynamic Route on their routing path.

## To Reproduce

Image we are trying to create a `hello world` app with support for i18n.
So I created the following structure:   

`pages -> [locale] -> hello.js`

(whereas "[locale]" is a directory)

if I go to '/en-gb/hello' I expect to see a message in english or 
if I go to '/pt-br/hello' I expect to see a message in portuguese

**Important to note**

- I'm providing all the possible language values through `getStaticPaths()` method.
- I'm providing the messages through `getStaticProps()` which depends only of the locale param value.
- I can export the application normally to static files and I have tested it using a simple `http server` and worked normally. 

## Screenshots
**next export output**
<img width="964" alt="Screenshot 2020-08-26 at 18 49 19" src="https://user-images.githubusercontent.com/3733116/91347711-af68f700-e7da-11ea-95b7-6a66e49b3b48.png">

**the code**
<img width="622" alt="Screenshot 2020-08-26 at 20 34 36" src="https://user-images.githubusercontent.com/3733116/91348424-a3316980-e7db-11ea-8877-ff8e6977a6fc.png">

**Static Optimization Indicator not appearing**
<img width="678" alt="Screenshot 2020-08-26 at 20 34 12" src="https://user-images.githubusercontent.com/3733116/91348473-b6443980-e7db-11ea-9f91-f145d887c3cf.png">

I know this is not very important, I just want to make sure I have understood it correctly.
Thanks.

**Comment by timneutkens** [comment, marked as answer]
In this case you're explicitly making it static so we don't show an indicator. When it is implicitly static the indicator is shown in order to make sure users are not confused about the behavior.

**Reply by nickmarca** [reply]
Make sense. Thanks 👍
